src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -176,10 +176,66 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    @FXML
+    BorderPane quotaBlock;
+    @FXML
+    Label quota;
+    @FXML
+    Label provider;
+    @FXML
+    Label quotaDescription;
+    @Inject
+    BoxClient boxClient;
```

Comment:
Why did you move these variables down?
Default (=expected) structure of a class is
```
class signature
static variables
instance variables
constructor(s)
methods
```
isn't it?